clojure-ts-mode.el

```diff
@@ -882,22 +884,87 @@ The returned value is expected to be the same as
 `clojure-get-indent-function' from `clojure-mode' for compatibility
 reasons.")
 
+(defun clojure-ts--unnest-dynamic-spec (spec current-depth)
+  "Recursively unnest SPEC, incrementally increasing the CURRENT-DEPTH.
```

Comment:
I'm pretty sure "unnest" doesn't exist. :D

Reply:
What do you think about `unpack`? :)

Reply:
Yeah, that's fine. `unfold` or `unwrap` will probably work fine as well.

Reply:
Renamed to unwrap. Thanks!